### ISSUE TITLE
Libtbb not found

### ISSUE THREAD

**Issue by ooreilly** [AMDResearch/omnitrace#148]
I did a binary rewrite for my executable, and tried running it via
```
srun -n 1 ./job.sh
```
in job.sh, I put
```
module use $OMNITRACE/share/modulefiles
module load omnitrace
omni_hip
```
I got this error message:
```
omni_hip: error while loading shared libraries: libtbb.so.2: can      not open shared object file: No such file or directory
```
To workaround the issue, I have to add the following to `job.sh` before calling the executable:
```
export LD_LIBRARY_PATH=/$OMNITRACE/lib/omnitrace:$LD_LIBRARY_PATH
```

**Comment by ooreilly** [comment]
I am using the binary release: `omnitrace-1.4.0-opensuse-15.3-ROCm-50200-PAPI-OMPT-Python3` 
```
module show module show omnitrace
------------------------------------------------------------------------------------------------------------------------------------
   /ccs/home/ooreilly/omnitrace-1.4.0-opensuse-15.3-ROCm-50200-PAPI-OMPT-Python3/share/modulefiles/omnitrace/1.4.0:
------------------------------------------------------------------------------------------------------------------------------------
whatis("omnitrace (version 1.4.0) ")
prepend_path("CMAKE_PREFIX_PATH","/autofs/nccs-svm1_home1/ooreilly/omnitrace-1.4.0-opensuse-15.3-ROCm-50200-PAPI-OMPT-Python3")
prepend_path("PATH","/autofs/nccs-svm1_home1/ooreilly/omnitrace-1.4.0-opensuse-15.3-ROCm-50200-PAPI-OMPT-Python3/bin")
prepend_path("LD_LIBRARY_PATH","/autofs/nccs-svm1_home1/ooreilly/omnitrace-1.4.0-opensuse-15.3-ROCm-50200-PAPI-OMPT-Python3/lib")
prepend_path("PYTHONPATH","/autofs/nccs-svm1_home1/ooreilly/omnitrace-1.4.0-opensuse-15.3-ROCm-50200-PAPI-OMPT-Python3/lib/python/site-packages")
setenv("omnitrace_DIR","/autofs/nccs-svm1_home1/ooreilly/omnitrace-1.4.0-opensuse-15.3-ROCm-50200-PAPI-OMPT-Python3/share/cmake/omnitrace")
help([[Loads omnitrace v1.4.0
]])
```

**Comment by jrmadsen** [comment]
Thanks for reporting this. libomnitrace-rt was missing a `$ORIGIN/omnitrace` rpath to find that library. It was fixed as part of #140 and will be in the v1.5.0 release.
